examples/build_model/conf/stack/base_decoder.yaml

```diff
@@ -0,0 +1,27 @@
+# base encoder settings that can be extended and overriden
+# we leave out the attention part for other config to override
+
+_target_: xformers.factory.block_factory.xFormerDecoderConfig
+reversible: False  # Optionally make these layers reversible to save memory
+num_layers: 3  # Optional this means that this config will repeat N times
+block_type: decoder
+dim_model: ${emb}
```

Comment:
I'm new to Hydra, but this means that it will be inferred from some broader context ?

Reply:
yes, the interpolation here is absolute - meaning it will interpolate the `emb` in the primary config file - `examples/build_model/conf/config.yaml`. Maybe it is not the best/most obvious way to config this. This will be better supported once Hydra supports partial instantiation, which should be coming soon in the latest release.